media.graphql

```diff
@@ -0,0 +1,185 @@
+# This GraphQL schema defines the META Media types and queries.
+#
+# It consists of the following main entities:
+#
+# MusicPerformer
+# MusicComposer
+# RecordLabel
+# MusicPublisher
+# MusicRecording
+# MusicWork
+# MusicRelease
+# MusicProduct
+#
+# Each entity has a set of widely used identifiers which are used to uniquely
+# identify them:
+#
+# ISNI - International Standard Name Identifier
+# IPI  - Interested Parties Information
+# DPID - DDEX Party Identifier
+# ISRC - International Standard Recording Code
+# ISWC - International Standard Musical Work Code
+# GRid - Global Release Identifier
+# UPC  - Universal Product Code
+#
+# Other entity properties (like performer name or recording title) are linked
+# to these identifiers by various sources in the META network.
+
+#
+# --- Main Entities ---
+#
+type MusicPerformer {
+  identifiers: [PartyIdentifier]
+
+  name: StringValue
+
+  products:   [MusicProductLink]
+  releases:   [MusicReleaseLink]
+  recordings: [MusicRecordingLink]
+  labels:     [RecordLabelLink]
+}
+
+type MusicComposer {
+  identifiers: [PartyIdentifier]
+
+  name: StringValue
+
+  works:      [MusicWorkLink]
+  publishers: [MusicPublisherLink]
+}
+
+type RecordLabel {
+  identifiers: [PartyIdentifier]
+
+  name: StringValue
+
+  products:   [MusicProductLink]
+  releases:   [MusicReleaseLink]
+  recordings: [MusicRecordingLink]
+  performers: [MusicPerformerLink]
+}
+
+type MusicPublisher {
+  identifiers: [PartyIdentifier]
+
+  name: StringValue
+
+  composers: [MusicComposerLink]
+  works:     [MusicWorkLink]
+}
+
+type MusicRecording {
```

Comment:
This cover both sound recordings and music video recordings.right ?
as ISRC defined both.

Reply:
We are just dealing with sound recordings currently, I think music video recordings would need to be a separate type, but I'm unsure until we have music video recording data.

Reply:
I am aware of two ERNs containing a video release, not sure of the contents of the latest batch though. They have very similar fields though, so it should be ok.